system/lib/libc/pthread_sigmask.c

```diff
@@ -0,0 +1,63 @@
+#define _GNU_SOURCE
```

Comment:
where is this code from?

Reply:
This is needed in order to get `sigandset` and `sigorset`: https://github.com/emscripten-core/emscripten/blob/2a71efcb2b41309ee3ac9452e79d893b87ff4206/system/lib/libc/musl/include/signal.h#L249-L254

Reply:
Where is it from, though? Is this musl code we didn't bundle before?

Reply:
Oh, I wrote this file myself.   The set of current handlers and blocked signals is all stored in the kernel itself so this is not code that normally needs to exist in a C library.

Reply:
Added copyright